### PR TITLE
label_studio_integration

### PR DESCRIPTION
**Pull request by Andrii-Sheba**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMDet3D.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: run_docker_mmseg.sh [hunk 1]
```diff
@@ -50,19 +57,29 @@ PRETRAIN_DIR="${PRETRAIN_DIR%%/}"
...
+    RUN_MODE="-t"
+fi
+
+
 #python3 -m tools.train configs/amr_segmentation/vit_uper.py
 
 #-d --restart=unless-stopped \
 docker run \
-  -it \
+  $RUN_MODE \
```
Comment (balthasarschuess) [inline run_docker_mmseg.sh:85]: Is this additional flag actually necessary?
During my testing. I was able to just provide the training command and the docker would shutdown properly afterwards.
Where is the advantage of adding this flag?

File: run_docker_mmdeploy.sh [hunk 2]
```diff
@@ -2,7 +2,48 @@
 
 CONTAINER_NAME="mmdeploy"
 IMAGE_TAG=mmdeploy:1.3.0
-docker build -t "$IMAGE_TAG" -f deploy_docker/Dockerfile . --progress=plain
+BUILD=true
+INTERACTIVE=true
+COMMAND="bash"
```
Comment (balthasarschuess) [inline run_docker_mmdeploy.sh]: Thanks for the fix here! I appreciate that you extended the script to work with arbitrary input commands.

File: run_docker_mmdeploy.sh [hunk 3]
```diff
@@ -2,7 +2,48 @@
...
+    fi  
+    exit 0
+}
+
+opts="b:i:c:h"
+while getopts "$opts" flag; do 
+  case "${flag}" in 
+    b) BUILD="$OPTARG" ;;
+    i) INTERACTIVE="$OPTARG" ;;
+    c) COMMAND=$OPTARG ;;
```
Comment (balthasarschuess) [inline run_docker_mmdeploy.sh]: I suggest you to:
- ditch this `COMMAND` parameter
- pass "$@" to docker directly instead
- remove the bash default and update the documentation instead.

The main reasoning behind this is, that there are differences between passing the command as a single command token and "$@" (which basically extends the arguments into several tokens while keeping escaping mechanics consistent).

File: run_docker_mmseg.sh (hunk 1 above)
Comment (Andrii-Sheba) [reply run_docker_mmseg.sh:85]: as we launch train/export docker container from webhooks and we don`t need interactive input I think we need it here.

File: run_docker_mmdeploy.sh (hunk 3 above)
Comment (Andrii-Sheba) [reply run_docker_mmdeploy.sh]: thanks, done!)

**Pull request merged**